javascripts/discourse/components/meta-banner.js

```diff
@@ -7,10 +7,11 @@ export default Component.extend({
 
   @discourseComputed("currentUser")
   shouldShow(currentUser) {
-    const isStaff = currentUser && currentUser.staff;
-    const lowTrustLevel = currentUser && currentUser.trust_level < 2;
-    // show banner only for anons and < TL 2
-    return !isStaff && (!currentUser || lowTrustLevel);
+    // const isStaff = currentUser && currentUser.staff;
+    // const lowTrustLevel = currentUser && currentUser.trust_level < 2;
+    // // show banner only for anons and < TL 2
+    // return !isStaff && (!currentUser || lowTrustLevel);
+    return true;
```

Comment:
Is this change intentional? It looks to be for debugging purposes.

Reply:
Good catch!